engine.json

```diff
@@ -6,7 +6,7 @@
     "O3DECopyrightYear": 2021,
     "O3DEBuildNumber": 0,
     "external_subdirectories": [
-        "Gems/RecastNavigation",
+	    "Gems/RecastNavigation",
```

Comment:
Undo this whitespace change which adds a tab

Reply:
fixed, tnx.